src/app/pages/BlocksPage/index.tsx

```diff
@@ -52,27 +72,52 @@ export const BlocksPage: FC = () => {
             },
           }
         },
+        keepPreviousData: tableView === TableLayout.Vertical,
       },
     },
   )
 
   return (
-    <PageLayout>
+    <PageLayout
+      mobileFooterAction={
+        tableView === TableLayout.Vertical && (
+          <LoadMoreButton pagination={pagination} isLoading={blocksQuery.isLoading} />
+        )
+      }
+    >
       {!isMobile && <Divider variant="layout" />}
-      <SubPageCard title={t('blocks.latest')}>
-        <Blocks
-          isLoading={blocksQuery.isLoading}
-          blocks={blocksQuery.data?.data.blocks}
-          limit={PAGE_SIZE}
-          verbose={true}
-          pagination={{
-            selectedPage: pagination.selectedPage,
-            linkToPage: pagination.linkToPage,
-            totalCount: blocksQuery.data?.data.total_count,
-            isTotalCountClipped: blocksQuery.data?.data.is_total_count_clipped,
-            rowsPerPage: NUMBER_OF_ITEMS_ON_SEPARATE_PAGE,
-          }}
-        />
+      <SubPageCard
+        title={t('blocks.latest')}
+        action={isMobile && <TableLayoutButton tableView={tableView} setTableView={setTableView} />}
+        noPadding={tableView === TableLayout.Vertical}
```

Comment:
Reason for wrapping the vertical blocks with Card is that header and action stay the same. @lukaw3d

Reply:
Visual bug is now at the top:
![image](https://user-images.githubusercontent.com/3758846/230242635-a9009ca9-a0c9-42d0-ac7a-e95d453e05c1.png)